import-export-cli/integration/devFirst_test.go

```diff
@@ -38,73 +39,76 @@ func TestInitializeProject(t *testing.T) {
 		initFlag: projectName,
 	}
 
+	validateInitializeProject(t, args)
+	//Remove Created project and logout
+	base.RemoveDir(projectName)
```

Comment:
Can we move this _base.RemoveDir()_  into the _validateInitializeProject()_ function? It is an essential cleanup step when running this scenario but it is a low level detail that is not required to be known at the high level of the test case.